runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -296,6 +295,13 @@ protected void initChannel(@Nonnull Channel ch) throws Exception {
                 ch.pipeline().addLast(new NettyCorfuMessageEncoder());
                 ch.pipeline().addLast(new ClientHandshakeHandler(parameters.getClientId(),
                     node.getNodeId(), parameters.getHandshakeTimeout()));
+
+                // If parameters include message filters, add corresponding filter handler
+                if (parameters.getNettyClientInboundMsgFilters() != null) {
+                    final InboundMsgFilterHandler inboundMsgFilterHandler =
+                            new InboundMsgFilterHandler(parameters.getNettyClientInboundMsgFilters());
+                    ch.pipeline().addLast(inboundMsgFilterHandler);
```

Comment:
A lot of our use cases to add and remove rules rely on the fact that we can add rules on Runtime.
For example: I would like my client to write to a stream the first time but then I would like to block all subsequent writes. The way to achieve this would be to initialize the list with an empty array and then add my rule to block in the same (empty) list. If I reinitialize the list, it will not reflect this rule in the client router.
Is this correct?